grafana-plugin/package.json

```diff
@@ -7,26 +7,27 @@
     "lint:fix": "eslint --fix --cache --ext .js,.jsx,.ts,.tsx --quiet ./src ./e2e-tests",
     "stylelint": "stylelint ./src/**/*.{css,scss,module.css,module.scss}",
     "stylelint:fix": "stylelint --fix ./src/**/*.{css,scss,module.css,module.scss}",
-    "build": "grafana-toolkit plugin:build",
-    "build:dev": "grafana-toolkit plugin:build --skipTest --skipLint",
+    "build": "webpack -c ./webpack.config.ts --env production",
+    "build:dev": "webpack -w -c ./webpack.config.ts --env development",
     "labels:link": "yarn --cwd ../../gops-labels/frontend link && yarn link \"@grafana/labels\" && yarn --cwd ../../gops-labels/frontend watch",
     "labels:unlink": "yarn --cwd ../../gops-labels/frontend unlink",
     "test": "jest --verbose",
     "test:silent": "jest --silent",
     "test:e2e": "yarn playwright test --grep-invert @expensive",
     "test:e2e-expensive": "yarn playwright test --grep @expensive",
+    "e2e": "yarn exec cypress install && yarn exec grafana-e2e run",
+    "e2e:update": "yarn exec cypress install && yarn exec grafana-e2e run --update-screenshots",
     "test:e2e:watch": "yarn test:e2e --ui",
     "test:e2e:gen": "yarn playwright codegen http://localhost:3000",
     "e2e-show-report": "yarn playwright show-report",
     "generate-types": "cd ./src/network/oncall-api/types-generator && yarn generate",
-    "dev": "grafana-toolkit plugin:dev",
-    "watch": "grafana-toolkit plugin:dev --watch",
-    "sign": "grafana-toolkit plugin:sign",
-    "ci-build:finish": "grafana-toolkit plugin:ci-build --finish",
-    "ci-package": "grafana-toolkit plugin:ci-package",
-    "ci-report": "grafana-toolkit plugin:ci-report",
+    "dev": "webpack -c ./.config/webpack/webpack.config.ts --env development",
+    "watch": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
+    "sign": "npx --yes @grafana/sign-plugin@latest",
     "start": "yarn watch",
     "plop": "plop",
+    "test:ci": "jest --passWithNoTests --maxWorkers 4",
+    "typecheck": "tsc --noEmit",
```

Comment:
do we need these 2?
`test:ci` doesn't seem to be used part of CI
`typecheck` seems more like a job for webpack than for us to run :thinking:

Reply:
`typecheck` looks useful for me, let's keep it, and  I've removed `test:ci`